### DISCUSSION TITLE
Display of verbatim for codes in the "View Graph" Report

### DISCUSSION THREAD

**Discussion by apar5070** [ccbogel/QualCoder#921] [Q&A]
As I was working in the QualCoder, as I created list of codes, sub-category and category; I was unable to present the list of verbatim for each of the codes/sub-category/category. It could also be that we could have selective verbatim to support it. Is it doable in QualCoder???

**Comment by ccbogel** [comment]
I am not sure what you mean exactly.
In the view graph report there is a button (four coloured squares) to display the coded segments of text files - selecting the files, then the codes then the colour.
